float32 torso
bool torso_change
float32 right
bool right_change
float32 left
bool left_change